src/puppetlabs/puppetdb/config.clj

```diff
@@ -316,13 +317,20 @@
   (let [default-web-router-service
         {:puppetlabs.puppetdb.cli.services/puppetdb-service "/pdb/query"
          :puppetlabs.puppetdb.dashboard/dashboard-service "/pdb"
-         :puppetlabs.puppetdb.metrics/metrics-service "/metrics"}]
+         :puppetlabs.puppetdb.metrics/metrics-service "/metrics"}
+        bootstrap-cfg (-> (find-bootstrap-config config-data)
+                          slurp
+                          str/split-lines)
+        dashboard-redirect? (contains? (set bootstrap-cfg)
```

Comment:
My mistake I think the pattern i am thinking of is
`(some #{"puppetlabs..."} boostrap-cfg)` which is in theory better because you don't always traverse the whole list necessarily